### DISCUSSION TITLE
Can you add a function to clear room messages with just one click, like Telegram, by clicking to delete all messages

### DISCUSSION THREAD

**Discussion by 337077418** [element-hq/element-meta#1413] [Ideas]
Can you add a function to clear room messages with just one click, like Telegram, by clicking to delete all messages，

**Comment by eNkKVS4HxHc9jaq** [comment]
Yeah, I'm also missing this feature. Would love to see this implemented in Element.

**Comment by ghost** [comment]
+1 here, it's even more needed when you have many messages that are not able to decrypt
